Libraries/pbxbuild/Sources/Tool/InterfaceBuilderCommon.cpp

```diff
@@ -62,9 +62,11 @@ TargetedDeviceSetting(pbxsetting::Environment const &environment)
 std::vector<std::string> Tool::InterfaceBuilderCommon::
 DeploymentTargetArguments(pbxsetting::Environment const &environment)
 {
-    return {
-        "--minimum-deployment-target",
-        environment.resolve(environment.resolve("DEPLOYMENT_TARGET_SETTING_NAME")),
-    };
+    std::string min_deploy_target = environment.resolve(environment.resolve("DEPLOYMENT_TARGET_SETTING_NAME"));
+    if (min_deploy_target.empty()) {
+        return {};
```

Comment:
Well, it feels to me kind of odd to be putting that in here.
It's a two level dereference we are making here on DEPLOYMENT_TARGET_SETTING_NAME,
and after all, we shouldn't be checking for sdk validity here.
Maybe we could add kind of validity checker at the end of SDK loads?